data/slack-jackson-dto/src/main/kotlin/com/kreait/slack/api/contract/jackson/group/chat/MeMessage.kt

```diff
@@ -0,0 +1,40 @@
+package com.kreait.slack.api.contract.jackson.group.chat
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.fasterxml.jackson.annotation.JsonSubTypes
+import com.fasterxml.jackson.annotation.JsonTypeInfo
+import com.kreait.slack.api.contract.jackson.JacksonDataClass
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME,
+        include = JsonTypeInfo.As.PROPERTY,
+        property = "ok",
+        visible = true)
+@JsonSubTypes(
+        JsonSubTypes.Type(value = SuccessfulChatMeMessageResponse::class, name = "true"),
+        JsonSubTypes.Type(value = ErrorChatMeMessageResponse::class, name = "false")
+)
+@JacksonDataClass
+sealed class ChatMeMessageResponse constructor(@JsonProperty("ok") open val ok: Boolean)
+
+@JacksonDataClass
+data class SuccessfulChatMeMessageResponse constructor(override val ok: Boolean,
+                                                       @JsonProperty("channel") val channel: String,
+                                                       @JsonProperty("ts") val team: String)
```

Comment:
this value is a timestamp not a team
name it timestamp
and use Instant with @InstantToString annotation